chatty-cathy-server/src/main/java/org/server/ClientHandler.java

```diff
@@ -0,0 +1,59 @@
+package org.server;
+
+import org.common.Message;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.*;
+import java.net.*;
+
+public class ClientHandler implements Runnable {
+    private static final Logger logger = LoggerFactory.getLogger(ClientHandler.class);
+    private final Socket clientSocket;
+    private ObjectOutputStream out;
+    private final Server server;
+
+    public ClientHandler(Socket socket, Server server) {
+        this.clientSocket = socket;
+        this.server = server;
+    }
+
+    @Override
+    public void run() {
+        ObjectInputStream in;
+        try {
+            in = new ObjectInputStream(new BufferedInputStream(clientSocket.getInputStream()));
+            out = new ObjectOutputStream(clientSocket.getOutputStream());
+            Message message;
+
+            while (true) {
+                try {
+                    message = (Message) in.readObject();
+                    if (message.getMessage().equals("QUIT")) break;
+                    logger.info(message.getUsername() + ": " + message.getMessage());
+                    server.broadcast(message);
+                } catch (IOException | ClassNotFoundException e) {
+                    logger.error(String.valueOf(e));
+                }
+            }
+
+            in.close();
+            clientSocket.close();
+        } catch (IOException e) {
+            logger.error(String.valueOf(e));
+        }
```

Comment:
This uses the convential `e` however still misses the conventional method to get the stack trace too:
```suggestion
        } catch (IOException e) {
            logger.error(e);
        }
```